spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from collections.abc import Mapping
```

Comment:
This import will not work on Python 2.7. You need to find the equivalent import and check for the Python version here.

Reply:
Good detail didn't think of backward compatibility since Python 2.7 is reaching EOL. Still, I think its a good step, will resolve this.